core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/BaseResourceCreateRequest.java

```diff
@@ -27,73 +29,66 @@
 
 	protected static final long serialVersionUID = 1L;
 
-	// the new codesystem's ID, if not specified, it will be auto-generated
+	// the new ID, if not specified, it will be auto-generated
 	@NotEmpty
-	protected String id;
+	String id;
 	
 	@NotEmpty
-	protected String url;
+	String url;
 	
 	@NotEmpty
-	protected String title;
+	String title;
```

Comment:
Now they are package-private, which is one step ahead, but not the ideal solution. Use private fields, please! 😄